providers/fips/self_test.c

```diff
@@ -172,6 +172,64 @@ DEP_FINI_ATTRIBUTE void cleanup(void)
 }
 #endif
 
+/*
+ * We need an explicit HMAC-SHA-256 KAT even though it is also
+ * checked as part of the KDF KATs.  Refer IG 10.3.
+ */
+static const unsigned char hmac_kat_pt[] = {
```

Comment:
How were these values generated?

Reply:
No idea.  KeyPair did them.

Reply:
I do not think it matters.

Reply:
It only matters if they were generated as dog food.. Not a known KAT.

Reply:
ACVP produces myriad "unknown" KATs :grin: